### ISSUE TITLE
Website redirects don't look right with Nginx

### ISSUE THREAD

**Issue by iliajie** [virtualmin/virtualmin-gpl#712]
Jamie, I have noticed some odd looking `AutoDiscover` destination redirects with Nginx:

![image](https://github.com/virtualmin/virtualmin-gpl/assets/4426533/d8fbe11f-13f5-4da9-8f4f-b9c64dbd83c3)

![image](https://github.com/virtualmin/virtualmin-gpl/assets/4426533/338df4fb-63eb-41c2-9513-a5071df03856)

I mean this line:

```
/home/alma9-pro/public_html$scheme://$host/cgi-bin/autoconfig.cgi
```

.. looks kinda odd, isn't it?

The line in the Nginx config looks fine to me:

```
rewrite ^\Q/AutoDiscover/AutoDiscover.xml\E(.*) $scheme://$host/cgi-bin/autoconfig.cgi$1 break;
```

**Comment by iliajie** [comment]
Jamie, we need to fix this.

**Comment by iliajie** [comment]
I have found what's causing this.

On my Alma system, when I enabled **Mail Client Configuration** a long time ago, I had the record in the Nginx config:

```
rewrite ^\Q/AutoDiscover/AutoDiscover.xml\E(.*) $scheme://$host/cgi-bin/autoconfig.cgi$1 break;
```

while on Debian, where I enabled it just recently the record was:

```
rewrite ^\Q/AutoDiscover/AutoDiscover.xml\E(.*) $scheme://$host/cgi-bin/autoconfig.cgi$1 redirect;
```

Virtualmin Nginx module should be able to correctly handle rewrites ending with `break`. The fix is in [this PR](https://github.com/virtualmin/virtualmin-nginx/pull/65).
